django/core/handlers/asgi.py

```diff
@@ -206,7 +206,7 @@ async def process_request(request, send):
             # because it should not raise unexpected errors that would prevent
             # us from cancelling process_request().
             asyncio.create_task(self.listen_for_disconnect(receive)),
-            asyncio.create_task(process_request(request, send)),
+            asyncio.create_task(process_request(request, send, scope)),
```

Comment:
This would require an update to `process_request` to take the extra parameter.

Reply:
correct, I was converting from 5.0.1 and put this wrong, its the response that needs the scope